addons/binding/org.openhab.binding.russound/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,324 @@
+            <channel id="restore" typeId="button"><label>Restore Favorite</label><description>Restores the favorite (ON=system favorite, OFF=zone favorite)</description></channel>
+            <channel id="delete" typeId="button"><label>Delete Favorite</label><description>Deletes the favorite (ON=system favorite, OFF=zone favorite)</description></channel>
+        </channels>
+        
+        <config-description>
+            <parameter name="favorite" type="integer" min="1" max="2" required="true">
+                <label>Favorite ID</label>
+                <description>The favorite identifier</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <thing-type id="bankpreset">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bank"/>
+        </supported-bridge-type-refs>
+
+        <label>Russound Bank Preset</label>
+        <description>Preset (usually frequency) within a Bank for a Source (usually a tuner)</description>
+
+        <channels>
+            <channel id="name" typeId="readonlystring"><label>Favorite Name</label><description>The system favorite name</description></channel>
+            <channel id="valid" typeId="button"><label>Favorite Valid</label><description>If the favorite is valid or not</description></channel>
+        </channels>
+        
+        <config-description>
+            <parameter name="preset" type="integer" min="1" max="6" required="true">
+                <label>Preset ID</label>
+                <description>The preset identifier</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    
+
+    <thing-type id="zonepreset">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="zone"/>
+        </supported-bridge-type-refs>
+
+        <label>Russound Zone Presets</label>
+        <description>Zone Preset Commands to save/restore/delete presets for the zone</description>
+
+        <channels>
+            <channel id="save" typeId="button"><label>Save Favorite</label><description>Saves zone as the preset</description></channel>
+            <channel id="restore" typeId="button"><label>Restore Favorite</label><description>Restores the reset to the zone</description></channel>
+            <channel id="delete" typeId="button"><label>Delete Favorite</label><description>Deletes preset for the zone</description></channel>
+        </channels>
+        
+        <config-description>
+            <parameter name="preset" type="integer" min="1" max="36" required="true">
+                <label>Preset ID</label>
+                <description>The preset identifier (1-36 corresponds to banks 1-6, presets 1-6)</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="string">

```

Comment:
I strongly suggest to remove the types string, readonlystring and button.
Channel types should express a certain functionality - which can be very different for the different channels.
Reusing a type is really only meant for situations, where you have 1..n of the very same functionality, e.g. zones to control.